src/Panel/Panel.php

```diff
@@ -450,7 +450,7 @@ public static function setLanguage(): ?string
         $kirby = kirby();
 
         // language switcher
-        if ($kirby->options('languages')) {
+        if ($kirby->option('languages')) {
```

Comment:
@distantnative This change weirdly broke the tests. `options()` method doesn't accept any params and I fixed it. But i have no idea why fail? Do you have an idea?

Reply:
Maybe we should move this one to its own PR as it seems to be about an actual bug and not just code quality? Would be easier to look/work on it in its own PR.

Reply:
I've created own PR to look closer #3505